assets/src/application/services/toaster/useToaster.tsx

```diff
@@ -1,26 +1,17 @@
 import { useContext } from 'react';
-
-import { ToasterHash, ToasterHook } from './types';
+import uuidv4 from 'uuid/v4';
+import { ToasterHook } from './types';
 import useDismissToast from './useDismissToast';
 import useErrorToast from './useErrorToast';
 import useInfoToast from './useInfoToast';
 import useLoadingToast from './useLoadingToast';
 import useSuccessToast from './useSuccessToast';
 import { ToastContext } from '../context/ToastProvider';
 
-const hash: ToasterHash = (message): string => {
-	const msg = JSON.stringify(message);
-	return msg
-		.split('')
-		.reduce((a, b) => {
-			a = (a << 5) - a + b.charCodeAt(0);
-			return a & a;
-		}, 0)
-		.toString();
-};
-
 const useToaster = (): ToasterHook => {
 	const toaster = useContext(ToastContext);
+	const hash = uuidv4;
```

Comment:
`uuidv4` must be called to get the id - `uuidv4()`

Reply:
Updated https://github.com/eventespresso/event-espresso-core/pull/2408/commits/b08b78a3f0d9efa84e19ac3ee516a4db8432ee41